ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.js

```diff
@@ -63,9 +55,9 @@ export default class MessagesList extends Component {
           this.args.message.startTime = new Date(this.args.message.startTime);
```

Comment:
I was under the impression components cannot or should not modify arguments directly. You can pass back a value in a function to the parent and handle from there, but it feels like this new Date should wrap the arg in the parent and be passed down that way.